bramkouwen/Neatly-In-Motion
# Base page layout creation

## What does this PR do?
* It creates index.html 
* It creates the basic page layout for index.html

## How do you check this PR?
* Pull down the branch
* Check the added index.html file  
* Check the base page layout

## Screenshots 
None.
